### PR TITLE
Add Section "How to Migrate from a Bitnami MediaWiki Stack"

### PR DESCRIPTION
**Pull request by juan131**
Description of the change

Change in the README.md file in order to add a new section that helps users to Migrate from a Bitnami Stack to the Bitnami Mediawiki container.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -292,6 +292,89 @@ To backup your application data follow these steps:
...
+# How to migrate from a Bitnami Mediawiki Stack
+
+You can follow these steps in order to migrate it to this container:
+
+*Note:* You only need to backup/restore the directories *files* and *plugins* if they existed in the previous version.
+
+1. Export the data from your SOURCE installation: (assuming an installation in `/opt/bitnami` directory)
+
+  ```bash
+  $ mysqldump -u root -p bitnami_mediawiki > backup-mediawiki-database.sql
```
Comment (sameersbn) [inline README.md]: should be `mysqldump -u root -p bitnami_mediawiki > ~/backup-mediawiki-database.sql`

_added the `~/`_

File: README.md [hunk 2]
```diff
@@ -292,6 +292,89 @@ To backup your application data follow these steps:
...
+  $ tar cfz ~/backup-mediawiki-plugins.tar.gz plugins
+  ```
+
+2. Copy the backup files to your TARGET installation:
+
+  ```bash
+  $ scp ~/backup-mediawiki-* YOUR_USERNAME@TARGET_HOST:~
+  ```
+
+3. Create the Mediawiki Container as described in the section #How to use this Image (Using Docker Compose)
```
Comment (sameersbn) [inline README.md]: fix link -> [How to use this Image (Using Docker Compose)](#run-the-application-using-docker-compose)

File: README.md [hunk 3]
```diff
@@ -292,6 +292,89 @@ To backup your application data follow these steps:
...
+  $ docker-compose exec mediawiki nami stop apache
+  ```
+
+6. Obtain the password used by Mediawiki to access the database in order avoid reconfiguring it:
+
+  ```bash
+  $ docker-compose exec mediawiki bash -c 'cat /opt/bitnami/mediawiki/LocalSettings.php | grep wgDBpassword'  
+  ```
+  
+7. Restore the database backup: (replace ROOT_PASSWORD below with your MariaDB root password)
```
Comment (sameersbn) [inline README.md]: I don't see a reference to `ROOT_PASSWORD`, maybe you mean `PASSWORD_OBTAINED_IN_STEP_6`?

File: README.md [hunk 4]
```diff
@@ -292,6 +292,89 @@ To backup your application data follow these steps:
...
+
+  ```bash
+  $ cd ~
+  $ docker-compose exec mariadb mysql -u root
+  $ MariaDB [(none)]> drop database bitnami_mediawiki;
```
Comment (sameersbn) [inline README.md]: I'm not aware of the VM configuration, but would it make sense to grep the database name from the configuration as done to retrieve  the value of `wgDBpassword`?

File: README.md [hunk 5]
```diff
@@ -292,6 +292,89 @@ To backup your application data follow these steps:
...
+  $ gunzip -c ./backup-mediawiki-database.sql.gz | docker exec -i $(docker-compose ps -q mariadb) mysql -u root bitnami_mediawiki
+  ```
+
+8. Restore files/plugins from backup (optional):
+
+  ```bash
+  $ cat ./backup-mediawiki-files.tar.gz | docker-compose exec mediawiki bash -c 'cd /bitnami/mediawiki/ ; tar -xzvf -'
+  $ cat ./backup-mediawiki-plugins.tar.gz | docker-compose exec mediawiki bash -c 'cd /bitnami/mediawiki/vendor ; tar -xzvf -'
+  $ docker-compose exec mediawiki ln -s /bitnami/mediawiki/vendor/plugins /opt/bitnami/mediawiki/vendor/plugins 
+  $ docker-compose exec mediawiki ln -s /bitnami/mediawiki/files  /opt/bitnami/mediawiki/files
```
Comment (sameersbn) [inline README.md]: Won't these link only persist during the lifetime of the container? If the mediawiki container is re-created the user would need to execute these command again right?

Also are you sure of the `/opt/bitnami/mediawiki/files` path in the container?  In the container it appears that image file uploads are stored in the images directory

```
lrwxrwxrwx  1 root root     25 Apr 10 14:11 images -> /bitnami/mediawiki/images
```

File: README.md [hunk 6]
```diff
@@ -292,6 +292,89 @@ To backup your application data follow these steps:
...
+  ```
+
+9. Fix mediawiki directory permissions (optional):
```
Comment (sameersbn) [inline README.md]: mediawiki -> Mediawiki

File: README.md (hunk 5 above)
Comment (sameersbn) [inline README.md]: In addition I think generally the plugin configurations are store in the `LocalSettings.php` file, I believe we need to tell users  to copy over the required plugin configuration changes from the `LocalSettings.php` of the VM

File: README.md (hunk 4 above)
Comment (juan131) [inline README.md]: The VM configuration always set "bitnami_mediawiki" as the database name. It's a good feedback anyway.

File: README.md (hunk 3 above)
Comment (juan131) [inline README.md]: I forgot to add it in the commands that come after.

File: README.md (hunk 5 above)
Comment (juan131) [inline README.md]: I need to investigate more into this, please don't accept the PR yet.

**Pull request merged**